gcp/appengine/frontend3/src/templates/vulnerability.html

```diff
@@ -113,6 +146,10 @@ <h3 class="mdc-layout-grid__cell--span-3">
               {%- else -%}
               <dd>{{ affected.package.name }}</dd>
               {%- endif -%}
+              {%- if 'purl' in affected.package -%}
+              <dt>Purl</dt>
+              <dd>{{ affected.package.purl }}</dd>
```

Comment:
Can we also try make this a monospaced font to see how it looks?

Reply:
updated [5bd7474](https://github.com/google/osv.dev/commit/5bd7474f0b96b9e3a5cd008f65df07ddc33fc732)

Reply:
sample output:

<img width="1338" alt="Screenshot 2024-05-15 at 1 44 26 PM" src="https://github.com/google/osv.dev/assets/73332835/f7c90716-18e5-413e-8b29-6de3a87a8a49">